cmd/kots/cli/velero.go

```diff
@@ -611,8 +607,7 @@ func VeleroConfigureAzureCmd() *cobra.Command {
 				return errors.New("velero namespace not found")
 			}
 
-			// init containers are names differently starting in velero 1.6
-			if !veleroStatus.ContainsPlugin("velero-plugin-for-microsoft-azure") || !veleroStatus.ContainsPlugin("velero-velero-plugin-for-microsoft-azure") {
```

Comment:
is this no longer the case?

Reply:
I modified `ContainsPlugin` to search for the substring. Not sure how this was working in airgap with a custom registry namespace.